### PR TITLE
Zone.strong

### PR DESCRIPTION
**Pull request by floitschG**
The `dart:async` zone API is in the process of becoming strong-mode clean.
This patch fixes a breakage introduced by this change.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.8.1
+version: 1.8.2
```
Comment (kevmoo) [inline pubspec.yaml:10]: Does this use the new API? Need to bump SDK constraint?

**Review by kevmoo — APPROVED** [review]
Please update CHANGELOG to be clear that changes were added to support dart v2

File: pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply pubspec.yaml:10]: Instead, do `1.8.2-dev` for now. Use that DEP in the SDK. We can talk about releasing when we're ready to publish that SDK

Comment (floitschG) [reply pubspec.yaml:10]: done.

**Comment by floitschG** [comment]
done.

**Review by kevmoo — APPROVED** [review]

File: pubspec.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.8.1
+version: 1.8.2-dev
```
Comment (kevmoo) [inline pubspec.yaml]: We're getting ready to publish. Remove the `-dev`

**Review by kevmoo — APPROVED** [review]

File: pubspec.yaml (hunk 2 above)
Comment (keertip) [reply pubspec.yaml]: done.

**Comment by nex3** [comment]
Is this still compatible with the old Zone API? If not, we should update the SDK constraint.

**Comment by kevmoo** [comment]
We're pretty sure it is. Travis passed on 1.24, etc

**Pull request merged**